klippy/extras/tmc2209.py

```diff
@@ -0,0 +1,102 @@
+# TMC2209 configuration
+#
+# Copyright (C) 2019  Stephan Oelze stephan.oelze@gmail.com
+#
+# This file may be distributed under the terms of the GNU GPLv3 license.
+
+import tmc2208, tmc2130, tmc, tmc_uart
+TMC_FREQUENCY=12000000.
+
+Registers = dict(tmc2208.Registers)
+
+Registers.update({
+    "TCOOLTHRS":    0x14,
+    "COOLCONF":     0x42,
+    "SGTHRS":       0x40,
+    "SG_RESULT":    0x41
+})
+
+ReadRegisters = tmc2208.ReadRegisters
+ReadRegisters.append("SG_RESULT")
```

Comment:
This would change tmc2208.ReadRegisters.  Use `ReadRegisters = tmc2208.ReadRegisters + ["SG_RESULT"]` instead.

Reply:
i missed a dict here, fixed.